bin/spice/pkg/context/context.go

```diff
@@ -50,13 +50,15 @@ type RuntimeContext struct {
 	isCloud         bool
 	httpClient      *http.Client
 	apiKey          string
+	userAgent	    string
```

Comment:
Match the indentation with the other params (the linter should do this automatically)